### ISSUE TITLE
Refresh Steam data not working

### ISSUE THREAD

**Issue by clugg** [bugdone/headshotbox#213]
Hi,
Recently I've found that the refresh player data button doesn't work anymore (up until the last month, it was perfect). However, if I attempt to view an individual user (or match, or the user list on the front page) when the data is stale, the request works for those specific users only. I've tested manually by dumping all of the steamids stored in the SQLite database (3421 apparently, although I remember having at least 5000 in the "refreshed X players" console message) and attempting to query the API with them, which returns a 400 bad request.

My working theory is that requests to the Steam API have some fixed maximum length which is being exceeded with the calls headshotbox is making. I noticed that you are using a dependency called 'watt' to make the API requests and can't see any obvious splitting of requests in either watt nor headshotbox's code, although I'm not great at reading clojure.

Interestingly enough nothing shows up in the logs despite this line:
https://github.com/bugdone/headshotbox/blob/fe8e4d2e7a1fdb7bc6fa335458f3862c699a3684/src/hsbox/steamapi.clj#L16

`18-08-05 15:34:57 clug-PC INFO [hsbox.handler:148] - Refresh players steam info`

I also manually changed the timestamps of all steamids in the database to 0 to ensure that they were stale, but still nothing.
Thanks.

**Comment by bugdone** [comment]
The limit is 100 and it is documented. The code also uses 100 ([line 55](https://github.com/bugdone/headshotbox/blob/fe8e4d2e7a1fdb7bc6fa335458f3862c699a3684/src/hsbox/steamapi.clj#L55))
I never use the refresh button (because I generally leave hsbox open for hours).
It might be buggy, I'll investigate.

**Comment by clugg** [comment]
Unfortunately even auto updating doesn't occur (or work) for me - perhaps there's a problem on my end but as previously mentioned manual queries seem to work. The refresh button was working up until 17.2 I believe, but I don't see any diffs that would affect its function.

**Comment by ajohnsen** [comment]
I made this little script to force update all user with retries when i had problems with it updating.
You can try pasting this into your browser console when you're browsing your site.
https://gist.github.com/ajohnsen/21f7a649221694df47d0b05e519d96bc

My problem with it not updating was that i was getting connection reset when contacting steamcommunity, this was caused by a setting on my router that ended up dropping packets because it though they were corrupt.

**Comment by clugg** [comment]
@ajohnsen 's script worked up until a certain point, but despite having a fresh database (deleted headshotbox.sqlite and processed all demos again), it seems like one of the steamid64s may be causing an issue. Once the error occurs the first time, it repeats forever, presumably on the same set of steamids.

```18-08-14 00:06:28 clug-PC DEBUG [hsbox.steamapi:57] - Getting fresh data from the API for 42 steamids
18-08-14 00:06:31 clug-PC DEBUG [hsbox.steamapi:57] - Getting fresh data from the API for 74 steamids
18-08-14 00:06:34 clug-PC DEBUG [hsbox.steamapi:57] - Getting fresh data from the API for 81 steamids
18-08-14 00:06:37 clug-PC DEBUG [hsbox.steamapi:57] - Getting fresh data from the API for 69 steamids
18-08-14 00:06:40 clug-PC DEBUG [hsbox.steamapi:57] - Getting fresh data from the API for 100 steamids
18-08-14 00:06:43 clug-PC DEBUG [hsbox.steamapi:57] - Getting fresh data from the API for 26 steamids
18-08-14 00:06:46 clug-PC DEBUG [hsbox.steamapi:57] - Getting fresh data from the API for 100 steamids
18-08-14 00:06:49 clug-PC DEBUG [hsbox.steamapi:57] - Getting fresh data from the API for 100 steamids
18-08-14 00:06:52 clug-PC DEBUG [hsbox.steamapi:57] - Getting fresh data from the API for 56 steamids
18-08-14 00:06:55 clug-PC DEBUG [hsbox.steamapi:57] - Getting fresh data from the API for 83 steamids
18-08-14 00:06:58 clug-PC DEBUG [hsbox.steamapi:57] - Getting fresh data from the API for 100 steamids
18-08-14 00:07:01 clug-PC DEBUG [hsbox.steamapi:57] - Getting fresh data from the API for 100 steamids
18-08-14 00:07:04 clug-PC DEBUG [hsbox.steamapi:57] - Getting fresh data from the API for 100 steamids
18-08-14 00:07:07 clug-PC DEBUG [hsbox.steamapi:57] - Getting fresh data from the API for 100 steamids
18-08-14 00:07:10 clug-PC DEBUG [hsbox.steamapi:57] - Getting fresh data from the API for 100 steamids
18-08-14 00:07:13 clug-PC DEBUG [hsbox.steamapi:57] - Getting fresh data from the API for 100 steamids
18-08-14 00:07:16 clug-PC DEBUG [hsbox.steamapi:57] - Getting fresh data from the API for 66 steamids
18-08-14 00:07:19 clug-PC DEBUG [hsbox.steamapi:57] - Getting fresh data from the API for 1 steamids
18-08-14 00:07:22 clug-PC ERROR [hsbox.handler:223] - 
                                                                     [37mjava.lang.Thread.run[m  [32m                             [m
                                     [37morg.eclipse.jetty.util.thread.QueuedThreadPool$3.run[m  [32m  QueuedThreadPool.java:  555[m
                                    [37morg.eclipse.jetty.util.thread.QueuedThreadPool.runJob[m  [32m  QueuedThreadPool.java:  635[m
                                            [37morg.eclipse.jetty.io.AbstractConnection$2.run[m  [32mAbstractConnection.java:  540[m
                                       [37morg.eclipse.jetty.server.HttpConnection.onFillable[m  [32m    HttpConnection.java:  257[m
                                              [37morg.eclipse.jetty.server.HttpChannel.handle[m  [32m       HttpChannel.java:  310[m
                                                   [37morg.eclipse.jetty.server.Server.handle[m  [32m            Server.java:  497[m
                                   [37morg.eclipse.jetty.server.handler.HandlerWrapper.handle[m  [32m    HandlerWrapper.java:   97[m
[37mring.adapter.jetty.proxy$org.eclipse.jetty.server.handler.AbstractHandler$ff19274a.handle[m  [32m                             [m
                                                      [33mring.adapter.jetty/proxy-handler/[1;33mfn[m  [32m              jetty.clj:   24[m
                                                  [33mring.middleware.session/wrap-session/[1;33mfn[m  [32m            session.clj:  103[m
                                                      [33mring.middleware.flash/wrap-flash/[1;33mfn[m  [32m              flash.clj:   35[m
                                [33mring.middleware.multipart-params/wrap-multipart-params/[1;33mfn[m  [32m   multipart_params.clj:  133[m
                                                    [33mring.middleware.params/wrap-params/[1;33mfn[m  [32m             params.clj:   64[m
                                      [33mring.middleware.nested-params/wrap-nested-params/[1;33mfn[m  [32m      nested_params.clj:   86[m
                                    [33mring.middleware.keyword-params/wrap-keyword-params/[1;33mfn[m  [32m     keyword_params.clj:   35[m
                                                      [33mhsbox.handler/wrap-cache-control/[1;33mfn[m  [32m            handler.clj:  228[m
                                                          [33mhsbox.handler/wrap-exception/[1;33mfn[m  [32m            handler.clj:  221[m
                                      [33mring.middleware.http-response/wrap-http-response/[1;33mfn[m  [32m      http_response.clj:    8[m
                                                                 [33mcompojure.core/routes/[1;33mfn[m  [32m               core.clj:  156[m
                                                                       [33mclojure.core/[1;33mapply[m  [32m               core.clj:  648[m
                                                                                      [37m...[m  [32m                             [m
                                                                   [33mcompojure.core/[1;33mrouting[m  [32m               core.clj:  148[m
                                                                   [33mcompojure.core/[1;33mrouting[m  [32m               core.clj:  151[m
                                                                        [33mclojure.core/[1;33msome[m  [32m               core.clj: 2592[m
                                                                [33mcompojure.core/routing/[1;33mfn[m  [32m               core.clj:  151[m
                                                             [33mcompojure.core/if-context/[1;33mfn[m  [32m               core.clj:  220[m
                                                                         [33mhsbox.handler/[1;33mfn[m  [32m            handler.clj:  205[m
                                                                                      [37m...[m  [32m                             [m
                                                                   [33mcompojure.core/[1;33mrouting[m  [32m               core.clj:  148[m
                                                                   [33mcompojure.core/[1;33mrouting[m  [32m               core.clj:  151[m
                                                                        [33mclojure.core/[1;33msome[m  [32m               core.clj: 2592[m
                                                                [33mcompojure.core/routing/[1;33mfn[m  [32m               core.clj:  151[m
                                                        [33mring.middleware.cors/wrap-cors/[1;33mfn[m  [32m               cors.clj:  156[m
                                                   [33mring.middleware.json/wrap-json-body/[1;33mfn[m  [32m               json.clj:   46[m
                                               [33mring.middleware.json/wrap-json-response/[1;33mfn[m  [32m               json.clj:   87[m
                                                                 [33mcompojure.core/routes/[1;33mfn[m  [32m               core.clj:  156[m
                                                                       [33mclojure.core/[1;33mapply[m  [32m               core.clj:  648[m
                                                                                      [37m...[m  [32m                             [m
                                                                   [33mcompojure.core/[1;33mrouting[m  [32m               core.clj:  148[m
                                                                   [33mcompojure.core/[1;33mrouting[m  [32m               core.clj:  151[m
                                                                        [33mclojure.core/[1;33msome[m  [32m               core.clj: 2592[m
                                                                [33mcompojure.core/routing/[1;33mfn[m  [32m               core.clj:  151[m
                                                             [33mcompojure.core/if-context/[1;33mfn[m  [32m               core.clj:  220[m
                                                                         [33mhsbox.handler/[1;33mfn[m  [32m            handler.clj:  132[m
                                                                                      [37m...[m  [32m                             [m
                                                                   [33mcompojure.core/[1;33mrouting[m  [32m               core.clj:  148[m
                                                                   [33mcompojure.core/[1;33mrouting[m  [32m               core.clj:  151[m
                                                                        [33mclojure.core/[1;33msome[m  [32m               core.clj: 2592[m
                                                                [33mcompojure.core/routing/[1;33mfn[m  [32m               core.clj:  151[m
                                                                                      [37m...[m  [32m                             [m
                                                                 [33mcompojure.core/routes/[1;33mfn[m  [32m               core.clj:  156[m
                                                                       [33mclojure.core/[1;33mapply[m  [32m               core.clj:  648[m
                                                                                      [37m...[m  [32m                             [m
                                                                   [33mcompojure.core/[1;33mrouting[m  [32m               core.clj:  148[m
                                                                   [33mcompojure.core/[1;33mrouting[m  [32m               core.clj:  151[m
                                                                        [33mclojure.core/[1;33msome[m  [32m               core.clj: 2592[m
                                                                [33mcompojure.core/routing/[1;33mfn[m  [32m               core.clj:  151[m
                                                              [33mcompojure.core/if-method/[1;33mfn[m  [32m               core.clj:   27[m
                                                               [33mcompojure.core/if-route/[1;33mfn[m  [32m               core.clj:   45[m
                                                        [33mcompojure.core/wrap-route-info/[1;33mfn[m  [32m               core.clj:  126[m
                                                  [33mcompojure.core/wrap-route-middleware/[1;33mfn[m  [32m               core.clj:  122[m
                                                             [33mcompojure.core/make-route/[1;33mfn[m  [32m               core.clj:  135[m
                                                                      [33mhsbox.handler/fn/[1;33mfn[m  [32m            handler.clj:  145[m
                                                                                      [37m...[m  [32m                             [m
                                                         [33mhsbox.steamapi/[1;33mget-steamids-info[m  [32m           steamapi.clj:   47[m
                                                         [33mhsbox.steamapi/[1;33mget-steamids-info[m  [32m           steamapi.clj:   55[m
                                                                       [33mclojure.core/[1;33mapply[m  [32m               core.clj:  641[m
                                                                         [33mclojure.core/[1;33mseq[m  [32m               core.clj:  137[m
                                                                                      [37m...[m  [32m                             [m
                                                                      [33mclojure.core/map/[1;33mfn[m  [32m               core.clj: 2646[m
                                                [33mhsbox.steamapi/[1;33mget-steamids-info-from-api[m  [32m           steamapi.clj:   29[m
                                                                      [33mclojure.core/[1;33mreduce[m  [32m               core.clj: 6545[m
                                                              [33mclojure.core.protocols/fn/[1;33mG[m  [32m          protocols.clj:   13[m
                                                                [33mclojure.core.protocols/[1;33mfn[m  [32m          protocols.clj:   75[m
                                                        [33mclojure.core.protocols/[1;33mseq-reduce[m  [32m          protocols.clj:   31[m
                                                              [33mclojure.core.protocols/fn/[1;33mG[m  [32m          protocols.clj:   19[m
                                                                [33mclojure.core.protocols/[1;33mfn[m  [32m          protocols.clj:  136[m
                                                                                      [37m...[m  [32m                             [m
                                             [33mhsbox.steamapi/get-steamids-info-from-api/[1;33mfn[m  [32m           steamapi.clj:   29[m
                                                                 [37mjava.lang.Long.parseLong[m  [32m                              (repeats 2 times)[m
[1;31mjava.lang.NumberFormatException[m: [3mnull[m
```

**Comment by ajohnsen** [comment]
Oh, then you’re having the same problem as me! The reason it’s failing is because one of the steam accounts has been deleted! So all @bugdone needs to do is to release the next version!  He fixed it in the latest commit. 

I have a modified update script I’ll post when I get to my computer that can help update all except that one that fails. :)

**Comment by bugdone** [comment]
Try 0.17.3 and see if it fixes it.
I'm curious if there are still problems when you don't use @ajohnsen's script

**Comment by clugg** [comment]
@bugdone I didn't have that error when using the refresh button, only when running the script @ajohnsen provided.

Updating to 17.3 hasn't fixed the refresh button, and using the manual script is still resulting in a java.lang.NumberFormatException, presumably for the same reasons as before. Logs for good measure:
```18-08-14 06:27:06 clug-PC INFO [hsbox.core:67] - HeadshotBox 0.17.3  
18-08-14 06:27:06 clug-PC INFO [hsbox.version:21] - Checking latest version on https://raw.githubusercontent.com/bugdone/headshotbox/master/latest-windows-version
18-08-14 06:27:06 clug-PC DEBUG [hsbox.version:23] - Latest version 0.17.2
18-08-14 06:27:10 clug-PC DEBUG [hsbox.indexer:138] - Indexer started
18-08-14 06:27:15 clug-PC INFO [hsbox.handler:148] - Refresh players steam info
18-08-14 06:27:43 clug-PC DEBUG [hsbox.steamapi:62] - Getting fresh data from the API for 1 steamids
18-08-14 06:27:46 clug-PC DEBUG [hsbox.steamapi:62] - Getting fresh data from the API for 1 steamids
18-08-14 06:27:49 clug-PC DEBUG [hsbox.steamapi:62] - Getting fresh data from the API for 2 steamids
18-08-14 06:27:52 clug-PC DEBUG [hsbox.steamapi:62] - Getting fresh data from the API for 1 steamids
18-08-14 06:27:55 clug-PC DEBUG [hsbox.steamapi:62] - Getting fresh data from the API for 1 steamids
18-08-14 06:27:57 clug-PC DEBUG [hsbox.steamapi:62] - Getting fresh data from the API for 1 steamids
18-08-14 06:28:00 clug-PC DEBUG [hsbox.steamapi:62] - Getting fresh data from the API for 59 steamids
18-08-14 06:28:03 clug-PC ERROR [hsbox.handler:223] - 
                                                                     [37mjava.lang.Thread.run[m  [32m                             [m
                                     [37morg.eclipse.jetty.util.thread.QueuedThreadPool$3.run[m  [32m  QueuedThreadPool.java:  555[m
                                    [37morg.eclipse.jetty.util.thread.QueuedThreadPool.runJob[m  [32m  QueuedThreadPool.java:  635[m
                                            [37morg.eclipse.jetty.io.AbstractConnection$2.run[m  [32mAbstractConnection.java:  540[m
                                       [37morg.eclipse.jetty.server.HttpConnection.onFillable[m  [32m    HttpConnection.java:  257[m
                                              [37morg.eclipse.jetty.server.HttpChannel.handle[m  [32m       HttpChannel.java:  310[m
                                                   [37morg.eclipse.jetty.server.Server.handle[m  [32m            Server.java:  497[m
                                   [37morg.eclipse.jetty.server.handler.HandlerWrapper.handle[m  [32m    HandlerWrapper.java:   97[m
[37mring.adapter.jetty.proxy$org.eclipse.jetty.server.handler.AbstractHandler$ff19274a.handle[m  [32m                             [m
                                                      [33mring.adapter.jetty/proxy-handler/[1;33mfn[m  [32m              jetty.clj:   24[m
                                                  [33mring.middleware.session/wrap-session/[1;33mfn[m  [32m            session.clj:  103[m
                                                      [33mring.middleware.flash/wrap-flash/[1;33mfn[m  [32m              flash.clj:   35[m
                                [33mring.middleware.multipart-params/wrap-multipart-params/[1;33mfn[m  [32m   multipart_params.clj:  133[m
                                                    [33mring.middleware.params/wrap-params/[1;33mfn[m  [32m             params.clj:   64[m
                                      [33mring.middleware.nested-params/wrap-nested-params/[1;33mfn[m  [32m      nested_params.clj:   86[m
                                    [33mring.middleware.keyword-params/wrap-keyword-params/[1;33mfn[m  [32m     keyword_params.clj:   35[m
                                                      [33mhsbox.handler/wrap-cache-control/[1;33mfn[m  [32m            handler.clj:  228[m
                                                          [33mhsbox.handler/wrap-exception/[1;33mfn[m  [32m            handler.clj:  221[m
                                      [33mring.middleware.http-response/wrap-http-response/[1;33mfn[m  [32m      http_response.clj:    8[m
                                                                 [33mcompojure.core/routes/[1;33mfn[m  [32m               core.clj:  156[m
                                                                       [33mclojure.core/[1;33mapply[m  [32m               core.clj:  648[m
                                                                                      [37m...[m  [32m                             [m
                                                                   [33mcompojure.core/[1;33mrouting[m  [32m               core.clj:  148[m
                                                                   [33mcompojure.core/[1;33mrouting[m  [32m               core.clj:  151[m
                                                                        [33mclojure.core/[1;33msome[m  [32m               core.clj: 2592[m
                                                                [33mcompojure.core/routing/[1;33mfn[m  [32m               core.clj:  151[m
                                                             [33mcompojure.core/if-context/[1;33mfn[m  [32m               core.clj:  220[m
                                                                         [33mhsbox.handler/[1;33mfn[m  [32m            handler.clj:  205[m
                                                                                      [37m...[m  [32m                             [m
                                                                   [33mcompojure.core/[1;33mrouting[m  [32m               core.clj:  148[m
                                                                   [33mcompojure.core/[1;33mrouting[m  [32m               core.clj:  151[m
                                                                        [33mclojure.core/[1;33msome[m  [32m               core.clj: 2592[m
                                                                [33mcompojure.core/routing/[1;33mfn[m  [32m               core.clj:  151[m
                                                        [33mring.middleware.cors/wrap-cors/[1;33mfn[m  [32m               cors.clj:  156[m
                                                   [33mring.middleware.json/wrap-json-body/[1;33mfn[m  [32m               json.clj:   46[m
                                               [33mring.middleware.json/wrap-json-response/[1;33mfn[m  [32m               json.clj:   87[m
                                                                 [33mcompojure.core/routes/[1;33mfn[m  [32m               core.clj:  156[m
                                                                       [33mclojure.core/[1;33mapply[m  [32m               core.clj:  648[m
                                                                                      [37m...[m  [32m                             [m
                                                                   [33mcompojure.core/[1;33mrouting[m  [32m               core.clj:  148[m
                                                                   [33mcompojure.core/[1;33mrouting[m  [32m               core.clj:  151[m
                                                                        [33mclojure.core/[1;33msome[m  [32m               core.clj: 2592[m
                                                                [33mcompojure.core/routing/[1;33mfn[m  [32m               core.clj:  151[m
                                                             [33mcompojure.core/if-context/[1;33mfn[m  [32m               core.clj:  220[m
                                                                         [33mhsbox.handler/[1;33mfn[m  [32m            handler.clj:  132[m
                                                                                      [37m...[m  [32m                             [m
                                                                   [33mcompojure.core/[1;33mrouting[m  [32m               core.clj:  148[m
                                                                   [33mcompojure.core/[1;33mrouting[m  [32m               core.clj:  151[m
                                                                        [33mclojure.core/[1;33msome[m  [32m               core.clj: 2592[m
                                                                [33mcompojure.core/routing/[1;33mfn[m  [32m               core.clj:  151[m
                                                                                      [37m...[m  [32m                             [m
                                                                 [33mcompojure.core/routes/[1;33mfn[m  [32m               core.clj:  156[m
                                                                       [33mclojure.core/[1;33mapply[m  [32m               core.clj:  648[m
                                                                                      [37m...[m  [32m                             [m
                                                                   [33mcompojure.core/[1;33mrouting[m  [32m               core.clj:  148[m
                                                                   [33mcompojure.core/[1;33mrouting[m  [32m               core.clj:  151[m
                                                                        [33mclojure.core/[1;33msome[m  [32m               core.clj: 2592[m
                                                                [33mcompojure.core/routing/[1;33mfn[m  [32m               core.clj:  151[m
                                                              [33mcompojure.core/if-method/[1;33mfn[m  [32m               core.clj:   27[m
                                                               [33mcompojure.core/if-route/[1;33mfn[m  [32m               core.clj:   45[m
                                                        [33mcompojure.core/wrap-route-info/[1;33mfn[m  [32m               core.clj:  126[m
                                                  [33mcompojure.core/wrap-route-middleware/[1;33mfn[m  [32m               core.clj:  122[m
                                                             [33mcompojure.core/make-route/[1;33mfn[m  [32m               core.clj:  135[m
                                                                      [33mhsbox.handler/fn/[1;33mfn[m  [32m            handler.clj:  145[m
                                                                                      [37m...[m  [32m                             [m
                                                         [33mhsbox.steamapi/[1;33mget-steamids-info[m  [32m           steamapi.clj:   52[m
                                                         [33mhsbox.steamapi/[1;33mget-steamids-info[m  [32m           steamapi.clj:   60[m
                                                                       [33mclojure.core/[1;33mapply[m  [32m               core.clj:  641[m
                                                                         [33mclojure.core/[1;33mseq[m  [32m               core.clj:  137[m
                                                                                      [37m...[m  [32m                             [m
                                                                      [33mclojure.core/map/[1;33mfn[m  [32m               core.clj: 2646[m
                                                [33mhsbox.steamapi/[1;33mget-steamids-info-from-api[m  [32m           steamapi.clj:   34[m
                                                                      [33mclojure.core/[1;33mreduce[m  [32m               core.clj: 6545[m
                                                              [33mclojure.core.protocols/fn/[1;33mG[m  [32m          protocols.clj:   13[m
                                                                [33mclojure.core.protocols/[1;33mfn[m  [32m          protocols.clj:   75[m
                                                        [33mclojure.core.protocols/[1;33mseq-reduce[m  [32m          protocols.clj:   31[m
                                                              [33mclojure.core.protocols/fn/[1;33mG[m  [32m          protocols.clj:   19[m
                                                                [33mclojure.core.protocols/[1;33mfn[m  [32m          protocols.clj:  136[m
                                                                                      [37m...[m  [32m                             [m
                                             [33mhsbox.steamapi/get-steamids-info-from-api/[1;33mfn[m  [32m           steamapi.clj:   34[m
                                                                 [37mjava.lang.Long.parseLong[m  [32m                              (repeats 2 times)[m
[1;31mjava.lang.NumberFormatException[m: [3mnull[m
```

**Comment by ajohnsen** [comment]
It seems like i might have the same problem still, and that NumberFormatException is a different problem. When i click the refresh steam data button nothing happens.

I can see in the network tab that the delete sent to api/steamids/info gets a 404, could that be the cause of the problem?

**Comment by clugg** [comment]
I also noticed the 404, but I ignored it because the headshotbox server was still logging that it was refreshing steam data, implying that it was receiving the request and returning a misleading status code.

**Comment by ajohnsen** [comment]
This issue is still not fixed, i have [this](http://hs.lolintel.com/api/steamids/info?steamids=76561198288857646,76561198098469890,76561198012722901,76561197963895637,76561198151344905,76561198110807321,76561197979497306,76561198113227791,76561198198539435,76561198075590933,76561197964746881,76561198282651158,76561198165008446,76561198088264429,76561198167770354,76561197985733716,76561198298609221,76561198121453111,76561197977071739,76561198312775596,76561197971016896,76561198042834978,76561198195020602,76561198160354066,76561198263182591,76561197973884005,76561197978088687,76561198206279476,76561198188520499,76561198041689148,76561198014755726,76561198017685327,76561198065462727,76561198217439664,76561198201453017,76561197984639557,76561198103389223,76561198287812537,76561197961170118,76561197990319716,76561197988133848,76561198122376956,76561198727101363,76561198138308300,76561198207414315,76561198364792701,76561198088199143,76561198061734727,76561198081267548,76561198136072205,76561197960329729,76561198054912578,76561198048256116,76561198019830840,76561198103148650,76561198031405522,76561197994861520,76561198082294011,76561198047928585,76561198020377617,76561198207155739,76561198069441512,76561197984523755,76561198080846561,76561198161544758,76561198081485102,76561198103474293,76561198114904257,76561198009610082,76561198132895984,76561198060322602,76561198800008437,76561198195514226,76561198068758491,76561197998563393,76561198120478779,76561198271331782,76561198044357671,76561198065920578,76561197986889211,76561198134656965,76561197982637012,76561197966940758,76561197965367341,76561198188665910,76561198073107878,76561197978469069,76561198306115523,76561198075635310,76561197960280679,76561197988559591,76561198326756038,76561198160769900,76561198118369110,76561198171028383,76561198079773984,76561198437156660,76561198364937905,76561198207660020,76561198279652094) call that still fails. 

In this one deleted profile is 76561198134656965,.

Eg calling [this](http://hs.lolintel.com/api/steamids/info?steamids=76561198134656965) properly return an empty object, but when i then add an existing profile to the same call it fails. [example](http://hs.lolintel.com/api/steamids/info?steamids=76561198134656965,76561198279652094)

**Comment by bugdone** [comment]
Thanks @ajohnsen. My previous attempt to fix #211 was bad.

I'm not sure if this issue is a duplicate or not of #211. If 0.17.4 fixes this issue, please close it.
